bears/general/GenericHelpers.py

```diff
@@ -0,0 +1,162 @@
+from coala_utils.string_processing.Core import unescaped_search_for
```

Comment:
Not sure about a name.... maybe just "RangeHelpers" or something?

And yes, maybe put the errors in an own file and we collect them there.
Makes things more tidy if everybody does that.